protostar-rust/cairo-tests/declare_test/tests/declare_test.cairo

```diff
@@ -0,0 +1,8 @@
+use result::ResultTrait;
+use protostar_print::PrintTrait;
+
+#[test]
+fn test_declare_simple() {
+    let class_hash = declare('simple_contract').unwrap();
+    assert(class_hash == 273, 'proper class hash');
```

Comment:
Please inlcude tests:
- multiple contracts in a single package
- declare of not-exisiting contract

Reply:
I do not think this is a top priority. I think a top priority is: to declare a simple contract, deploy it, and invoke/call. Only then I would add what you suggest.

Reply:
@karol-bisztyga I don't agree here, what we do is much more coplicated that the solution offloading compilation to Scarb. Tests can be done in a separate PR, but the compilation should be offloaded to scarb here. It will simplify things a lot in terms of complexity offloading a lot of stuff to scarb, which is a good thing

Reply:
If you think conflicts are problem, I am ok with introducing those changes in a separate PR but it should be done before anything else.

Reply:
ok let's defer to separate PRs

Reply:
I think there's a misunderstanding here: I only said that I think we should do the tests you mentioned later, not now. I agree we should move building to scarb. I only referred to the tests.

Reply:
Discussed offline, the last 2 comments of Maks are irrelevant, resolving.